### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by suraneti**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -61,19 +61,19 @@ In plain english, the general idea is that each oomd config one or more
...
   * `engine`: oomd engine logs
   * `plugins`: logs written by plugins
 
-## Runtime evaulation rules
+## Runtime evaluation rules
 
 * Every plugin must return CONTINUE or STOP.
   * CONTINUE
-    * For DETECTORs, noop
+    * For DETECTORs, loop
```
Comment (danobi) [inline docs/configuration.md]: This is supposed to be `noop`, short for no-operation: https://en.wikipedia.org/wiki/NOP_(code)

Comment (suraneti) [reply docs/configuration.md]: OK, I've updated following your request.

**Comment by danobi** [comment]
Thankss!
